extensions/amp-sidebar/1.0/toolbar.js

```diff
@@ -47,6 +50,9 @@ export class Toolbar {
     /** @private {Element|undefined} */
     this.targetElement_ = undefined;
 
+    /** @private {Element|undefined} */
+    this.placeholder_ = undefined;
```

Comment:
Let's talk offline about this as I cannot find it. 

and this placehodler is used for the elements that will reserve height for our toolbar.